Balloon/app/memstat.cpp

```diff
@@ -202,8 +205,25 @@ BOOL CMemStat::Update()
         m_Stats[idx].val = statex.ullTotalPhys;
         idx++;
 
+        status = memory->Get(
+                             L"CacheBytes",
+                             0,
+                             &var_val,
+                             NULL,
+                             NULL
+                             );
+
+        if (FAILED(status) || (var_val.vt == VT_NULL)) {
+            PrintMessage("Cannot get CacheBytes");
+            var_val.vt = 0;
```

Comment:
Ah, I see your point now.
You mean trying to use varint after marking vt as illegal. 
You  are right, we can get there if someone decided to call Get
with wrong parameters, like incorrect name of property, when GetSystemPropertyByName
call will fail, or any other random crap instead of third parameter . Fix to this potential problem can (and probably should) be submitted as a separate patch.